doc/.sphinx/_integration/lxd.html

```diff
@@ -0,0 +1,43 @@
+<header id="header" class="p-navigation">
```

Comment:
I agree, but I can't really think of a way that would be easier.
We could either add a bunch of variables to each config file and use them in the template, or add a script that generates the files for each doc set. Both is possible, but I'm not sure it would actually make things easier to understand.

Reply:
Given the circumstances I don't think this is a priority atm and you're likely right about the added complexity.